infrastructure/metrics/src/main/java/tech/pegasys/teku/infrastructure/metrics/SettableDoubleGauge.java

```diff
@@ -17,19 +17,19 @@
 import org.hyperledger.besu.plugin.services.MetricsSystem;
 import org.hyperledger.besu.plugin.services.metrics.MetricCategory;
 
-public class SettableGauge {
+public class SettableDoubleGauge {
```

Comment:
Looks like we don't need this rename anymore :D

Reply:
Yep :D